src/dlr.cc

```diff
@@ -129,6 +132,29 @@ int CreateDLRModelFromTFLite(DLRModelHandle *handle,
 }
 #endif // DLR_TFLITE
 
+#ifdef DLR_TENSORFLOW
+/*! \brief Translate c args from ctypes to std types for DLRModelFromTensorflow ctor.
+ */
+int CreateDLRModelFromTensorflow(DLRModelHandle *handle,
+                   const char *model_path,
```

Comment:
Allignment is off here. Use `clang-format --style=google -i src/dlr.cc`

Reply:
Let me fix this particular block and run `clang-format` for existing old files in a separate PR to make it clear what was actually changed.